### ISSUE TITLE
Backup archive file name

### ISSUE THREAD

**Issue by 2mik** [NikitaYakuntsev/ftp-backup#2]
Hi!
How archive file names are generated?

**Comment by NikitaYakuntsev** [comment]
Hello,
That's a [Unix timestamp](https://en.m.wikipedia.org/wiki/Unix_time). 
Actually for python that's a time since 1st January of the epoch, and for unix it starts in 1970. 
You can check when your epoch started using 
`from time import gmtime`
`gmtime(0)`.
For more information about module **time** please refer https://docs.python.org/2/library/time.html

**Comment by 2mik** [comment]
Thank you.
As for me it would be more convenient if file name can be readable by a human.

**Comment by NikitaYakuntsev** [comment]
The main idea of such a naming is to create monotonically increasing sequence to be able to extract files in the correct order. 
Unix timestamp has 2 functional advantages: 
1. timestamp's value always increases (only if system clock hasn't been set back)
2. it could be simply converted back to datetime value to let us know when was the archive created.

**Comment by 2mik** [comment]
That's right, but the same advantages has the name format yyyy-MM-dd-hh-mm-ss
It can be not unique if you create 2 archives in a one second, but it is a very rare situation.

**Comment by NikitaYakuntsev** [comment]
I think you can mark issues with custom "change request" label to let me know if you want to see this features in the next update. 
And I'm going to find out best practices of versioning and tagging to provide "honest" release.

**Comment by 2mik** [comment]
Probably you should create this label first. Label link is disabled for me.
I think you should do some promotion for your tool )
